Sources/Network/Connection/ProtocolConnectionFactory.swift

```diff
@@ -132,52 +132,86 @@ extension ProtocolConnectionFactory {
     }
 }
 
-class EmptyConsensusService : ConsensusService, ConnectionProtocol, ConsensusServiceConnection {
-    func proposeTx(_ tx: External_Tx, completion: @escaping (Result<ConsensusCommon_ProposeTxResponse, ConnectionError>) -> Void) {
+class EmptyConsensusService: ConsensusService, ConnectionProtocol, ConsensusServiceConnection {
+    func proposeTx(
+        _ tx: External_Tx,
+        completion: @escaping (Result<ConsensusCommon_ProposeTxResponse, ConnectionError>) -> Void
+    ) {
         logger.assertionFailure("Not Implemented")
     }
 }
 
-class EmptyBlockchainService : BlockchainService, ConnectionProtocol, BlockchainServiceConnection {
-    func getLastBlockInfo(completion: @escaping (Result<ConsensusCommon_LastBlockInfoResponse, ConnectionError>) -> Void) {
+typealias LastBlockInfoRespResult = Result<ConsensusCommon_LastBlockInfoResponse, ConnectionError>
+class EmptyBlockchainService: BlockchainService, ConnectionProtocol, BlockchainServiceConnection {
+    func getLastBlockInfo(
+        completion: @escaping (LastBlockInfoRespResult) -> Void
+    ) {
         logger.assertionFailure("Not Implemented")
     }
 }
 
-class EmptyFogViewService : FogViewService, ConnectionProtocol, FogViewServiceConnection {
-    func query(requestAad: FogView_QueryRequestAAD, request: FogView_QueryRequest, completion: @escaping (Result<FogView_QueryResponse, ConnectionError>) -> Void) {
+class EmptyFogViewService: FogViewService, ConnectionProtocol, FogViewServiceConnection {
+    func query(
+        requestAad: FogView_QueryRequestAAD,
+        request: FogView_QueryRequest,
+        completion: @escaping (Result<FogView_QueryResponse, ConnectionError>) -> Void
+    ) {
         logger.assertionFailure("Not Implemented")
     }
 }
 
-class EmptyFogMerkleProofService : FogMerkleProofService, ConnectionProtocol, FogMerkleProofServiceConnection {
-    func getOutputs(request: FogLedger_GetOutputsRequest, completion: @escaping (Result<FogLedger_GetOutputsResponse, ConnectionError>) -> Void) {
+class EmptyFogMerkleProofService: FogMerkleProofService,
+    ConnectionProtocol,
+    FogMerkleProofServiceConnection
+{
+    func getOutputs(
+            request: FogLedger_GetOutputsRequest,
+            completion: @escaping (Result<FogLedger_GetOutputsResponse, ConnectionError>) -> Void
```

Comment:
double indented params on declaration here, where others are single-indented.

```suggestion
        request: FogLedger_GetOutputsRequest,
        completion: @escaping (Result<FogLedger_GetOutputsResponse, ConnectionError>) -> Void
```